STYLE.md

```diff
@@ -0,0 +1,318 @@
+are for.
+
+# STYLE
+
+## HEADER
+
+Templates
+**always**
+start with a header
+
+	# Template file for '<pkgname>'
+
+If your package requires another package to have version synchrony a comment
+can be added below the header.
+
+	# Requires version and revision synchrony with '<pkgname>'
+
+## VARIABLES
+
+Immediately after begin the variables, it is required that
+**pkgname**,
+**version**
+then
+**revision**
+start the template unless
+**reverts**
+is used, which must come between
+**pkgname**
+and
+**version**.
+
+	pkgname=foo
+	reverts="1.0.1-rc2"
+	version=1.0.0
+	revision=1
+
+If
+**noarch**
+is to be used, it should come after
+**revision**
+
+If applicable
+**wrksrc**
+and
+**build\_wrksrc**
+come right after revision or noarch, then
+**build\_style**.
+Variables that pertain to a build\_style like
+**go\_import\_path**,
+**configure\_args**,
+**make\_install\_target**
+must come directly after it. and afterwards come the dependencies.
+
+Dependencies are ordered as
+**hostmakedepends**,
+**makedepends**,
+**depends**,
+**checkdepends**,
+
+	hostmakedepends="automake gettext-devel glib-devel gobject-introspection gtk-doc
+	 gtk-update-icon-cache pkg-config $(vopt_if wayland 'wayland-devel
+	 wayland-protocols')"
+	makedepends="at-spi2-atk-devel gdk-pixbuf-devel libepoxy-devel pango-devel
+	 $(vopt_if colord 'colord-devel') $(vopt_if cups 'cups-devel')
+	 $(vopt_if wayland 'libxkbcommon-devel wayland-devel wayland-protocols
+	 libwayland-egl MesaLib-devel') $(vopt_if x11 'libXcursor-devel libXdamage-devel
+	 libXext-devel libXinerama-devel libXi-devel libXrandr-devel
+	 libXcomposite-devel')"
+	depends="gtk-update-icon-cache shared-mime-info $(vopt_if x11 'dbus-x11')"
+
+After the dependencies come
+**short\_desc**,
+start it with a Uppercase character, don't pass over 72 chars.
+When writing
+**short\_desc**
+avoid starting with superfluous characters like A.
+
+	# Wrong!
+	# Superfluous usage of A
+	short_desc="A configuration language guaranteed to terminate"
+	# Correct!
+	short_desc="Configuration language guaranteed to terminate"
+
+**maintainer**
+follows.
+It uses a
+**name &lt;email&gt;**
+scheme, name and email must match the git equivalents used to commit.
+The email must be reachable.
+It is used in  case something needs to be done
+about the package that will be used for contact.
+
+Afterwards we have
+**license**
+which should be specified according to the SPDX short-identifier.
+If multiple licenses apply then separate them with a comma and a whitespace.
+
+	# Wrong!
+	# GPL-2 isn't a SPDX short-identifier
+	# there is no proper separation between the licenses
+	license="GPL-2 MPL-2.0"
+	# Correct!
+	license="LGPL-2.1-or-later, Apache-2.0"
+
+**homepage**,
+**changelog**
+and
+**distfiles**
+follow, if multiple distfiles are present then they should be broken with a
+newline, the same applies to
+**checksum**
+
+	homepage="https://foosoft.com"
```

Comment:
Please could you specify that the homepage should always be in **plain text**
I mean, not a substitution like `https://github.com/${pkgname}`
It's annoying.
Thank you.